tf-managed/live/environments/dev/applications/forest-calibnet/terragrunt.hcl

```diff
@@ -0,0 +1,14 @@
+# Automatically find the root terragrunt.hcl and inherit its
+# configuration
+include "root" {
+  path = find_in_parent_folders()
+}
+
+# Load the actual Terraform module
+terraform {
+  source = format("%s/../modules/forest-in-docker-in-droplet", get_parent_terragrunt_dir())
+}
+
+inputs = {
```

Comment:
Let's have it consistent with other services, and have the droplet size here explicitly. Same for the name of the service.